### DISCUSSION TITLE
Multiple default projects

### DISCUSSION THREAD

**Discussion by pluma9** [ankitpokhrel/jira-cli#611] [Ideas]
I work in multiple projects (nearly 8). I need to specify the projects in almost every jira command. Typing 8 projects in each command is impractical.

Should we take #182 one step further and allow the user to specify multiple default projects in the config file?

**Comment by ankitpokhrel** [comment]
Wondering if you can rely on shell variable or alias for this?

**Comment by gbaker-hpe** [comment]
Does an alias or a shell variable solve for querying multiple projects at the same time?  There are two use cases here that I can see.

1. Project managers often times will want to query multiple projects at the same time.  e.g. show me all of the newly created issues in these 7 projects.
2. Some very large projects span many Jira projects, so to get an accurate view of the work, multiple queries across multiple projects need to be made.

If nothing else, a command to quickly change the default project would be helpful.  This would effectively change the idea of a "default" project to an "active" project.

A good example of a CLI with this type of behavior is the [Google Public Cloud CLI.](https://cloud.google.com/sdk/gcloud)

The CLI supports [changing projects](https://cloud.google.com/sdk/gcloud/reference/config/set) using this syntax `gcloud config set project <PROJECT_ID>`.  I could see something similar working for the Jira CLI.  This would at least solve for quickly changing projects without having to use `-p` often, but it would not solve for the problem of wanting to query many projects at the same time when listing issues, for example.

**Comment by gbaker-hpe** [comment]
If there's a design that you're comfortable with as a project owner, I can ask around my team to see if anyone would be able to pick the work up as a side project and contribute a PR as a feature proposal.
